ui/package.json

```diff
@@ -55,7 +54,9 @@
     "@redux-devtools/extension": "^3.2.3",
     "@rjsf/core": "^5.0.0-beta.11",
     "@rjsf/material-ui": "^5.0.0-beta.11",
+    "@rjsf/utils": "^5.0.0-beta.14",
```

Comment:
let's stick to ^5.0.0-beta.11. [#ref](https://github.com/meshery/meshery/pull/6606#issuecomment-1338729367)

Reply:
@harkiratsm Wasn't intentional, just noticed that the package wasn't installed, I just ran the command to install the package itself, but I can pin this at ^5.0.0-beta.11.

Reply:
Thanks @acald-creator